framework/db/Command.php

```diff
@@ -537,7 +537,7 @@ public function createTable($table, $columns, $options = null)
     {
         $sql = $this->db->getQueryBuilder()->createTable($table, $columns, $options);
 
-        return $this->setSql($sql);
+        return $this->setSql($sql)->requireTableSchemaRefresh($table);
```

Comment:
If the table is just created, there should be no past cache so it makes no sense to refresh it.